.github/workflows/e2e-tests.yml

```diff
@@ -63,3 +63,19 @@ jobs:
         run: |
           export NIGHTLY_TOOLCHAIN=${{steps.toolchain.outputs.name}}
           ./scripts/e2e-tests.sh
+      # Coverage
+      - name: cargo install cargo-llvm-cov
```

Comment:
Can you explain how it should work?
Are we checking test coverage of test framework here or integration tests?
And if it is last one how it works?

Reply:
Ofc! So, this is running the `e2e` tests and collecting coverage data, which is uploaded to codecov.io and merged with the data uploaded from the `test` workflow.

> Are we checking test coverage of test framework here, unit tests code or contracts library?

Not entirely sure what you mean. We have two workflows right now that collect coverage data: `test` and this one (`e2e`).

* `test` runs `cargo test --features std`
* `e2e` runs `cargo test --features std,e2e --test '*'`

So, the first one runs unit tests, and the second one runs integration tests.

Does this answer your questions?

Reply:
I mean it in a way of how code coverage implemented inside. Is it changing binary code to insert counters or smth else? And will it work for wasm binaries deployed to nitro node?

Reply:
ah, no clue hehe

Reply:
I expect it to work, yes. for example, events are getting covered, but we need to write more tests and see how the diff changes.

Reply:
oke. Will check out then